api/src/main/java/ai/djl/repository/RepositoryFactoryImpl.java

```diff
@@ -53,6 +54,14 @@ public Repository newInstance(String name, String url) {
             return factory.newInstance(name, url);
         }
 
+        try {
+            if (!"file".equals(scheme)) {
+                uri.toURL();
```

Comment:
it looks like toURL() check the schema already, why do we still need line 58?

Reply:
We can accept raw file path: "/User/home/model", which will fail toURL() checking